main.tf

```diff
@@ -33,27 +33,43 @@ resource "aws_security_group" "default" {
   count       = module.this.enabled ? 1 : 0
   name        = module.this.id
   vpc_id      = var.vpc_id
-  description = "Bastion security group (only SSH inbound access is allowed)"
+  description = "Bastion host security group"
```

Comment:
And you went and did it anyway? https://github.com/cloudposse/terraform-aws-ec2-bastion-server/pull/29#discussion_r490435635